client/api.go

```diff
@@ -170,6 +172,10 @@ type API struct {
 	Transfers *transfer.Client
 	// UsageRecords is the client used to invoke usage record related APIs.
 	UsageRecords *usagerecord.Client
+	// RadarValueLists is the client used to invoke /radar/value_lists APIs.
+	RadarValueLists *valuelist.Client
+	// RadarValueListItems is the client used to invoke /radar/value_list_items APIs.
+	RadarValueListItems *valuelistitem.Client
```

Comment:
Sorry for the nit, but now that these are renamed, let's shift them up a bit to fall into alphabetical order (and same with where they're initialized below).

Reply:
Thanks, fixed!